sdk/azidentity/README.md

```diff
@@ -1,6 +1,10 @@
-# Azure Identity client library for Go
+# Azure Identity Client Module for Go
 
-The Azure Identity library provides a set of credential types for use with
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/azidentity)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity)
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/go/go%20-%20azidentity?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1846&branchName=master)
+[![Code Coverage](https://img.shields.io/azure-devops/coverage/azure-sdk/public/1846/master)](https://img.shields.io/azure-devops/coverage/azure-sdk/public/1846/master)
+
+The `azidentity` module provides a set of credential types for use with
```

Comment:
Since you have this open already could you change the "which" in this sentence to "that" for me please?

Reply:
done

Reply:
Thanks!